package.json

```diff
@@ -4,7 +4,6 @@
   "description": "S3 Compatible Cloud Storage client",
   "main": "./dist/main/minio.js",
   "module": "./dist/esm/minio.mjs",
-  "types": "./dist/main/minio.d.ts",
```

Comment:
I don't think deleting this one would solve the Typescript typings reference issues

Reply:
we have `./dist/esm/minio.d.mts` for `./dist/esm/minio.mjs` and `./dist/main/minio.d.ts` for `./dist/main/minio.js`, TypeScript will resolve to these types.

Reply:
I tested this locally and all `moduleResolution` works fine.